index.html

```diff
@@ -0,0 +1,178 @@
+                <img class="adornIcon taco" src="./images/iconsMxImages/6bigTaco-removebg-preview.png" alt="">
+                <img class="adornIcon hat" src="./images/iconsMxImages/2hat-removebg-preview.png" alt="">
+                <img class="adornIcon pinata" src="./images/iconsMxImages/3pin-removebg-preview.png" alt="">
+                <img class="adornIcon guacamole" src="./images/iconsMxImages/7guacamole-removebg-preview.png" alt="">
+                <!-- --- -->
+                <div id="textMain">
+                    <div id="helloAndEventContainer">
+                        <div id="hello">"Hello! Food Lovers"</div>
+                        <h1 id="event">INTERNATIONAL MEXICAN FOOD CONVENTION 2023</h1>
+                    </div>
+                    <div id="description">
+                        <p id="descriptionParagraph">A joyful celebration believing in the value of openness and
+                            sharing, creating a positive change with people from over 80 countries is taking place in
+                            October, in Korea.</p>
+                    </div>
+                    <div id="dateAndPlaceContainer">
+                        <div id="date">2023.10.15(THU) ~ 16(FRI)</div>
+                        <div id="place">@ National Museum of Karea, Art Center Nabi and more</div>
+                    </div>
+                </div>
+        </section>
+        <!-- Program Details Section -->
+        <section id="programDetails">
+            <div class="titleAndUnderlineContainer">
+                <h2 class="title">Main Program</h2>
+                <span class="underline"></span>
+            </div>
+            <div id="programListContainer">
+                <div class="programItems">
+                    <img class="programImages" src="./images/1programMusic.png" alt="">
+                    <h3 class="programNames">Music</h3>
+                    <p class="programParagraphs">Listen to the speakers from various countries about the messages of
+                        sharing and opening.</p>
+                </div>
+                <div class="programItems">
+                    <img class="programImages" src="./images/2programComedy.png" alt="">
+                    <h3 class="programNames">Comedy Show</h3>
+                    <p class="programParagraphs">Appreciate various creations applying CC license of artists, organized
+                        from Art Center Nabi.</p>
+                </div>
+                <div class="programItems">
+                    <img class="programImages" src="./images/3programLecture.png" alt="">
+                    <h3 class="programNames">Talks</h3>
+                    <p class="programParagraphs">Have the time to share your thoughts and opinions with experts for each
+                        topic.</p>
+                </div>
+                <div class="programItems">
+                    <img class="programImages" src="./images/4programPhoto.png" alt="">
```

Comment:
- [optional] kindly add `alt` descriptions to all your images. this helps where users of screen magnification may not be able to get a complete view of the image and thus need the alt tag to let them know what the purpose of the image is